internal/metrics/tags.go

```diff
@@ -0,0 +1,13 @@
+package metrics
+
+import (
+	"go.opencensus.io/tag"
+)
+
+var (
+	keyMethod, _      = tag.NewKey("method")
```

Comment:
Ignored err makes me nervous. Would it be possible to keep the declarations global but move the instantiation to an `err` checked function?

Reply:
Yep.  Done.  Wasn't too worried since they're static, but it's easy to do the right thing.